lib/apiservers/engine/backends/volume.go

```diff
@@ -198,7 +202,7 @@ func (v *Volume) VolumeRm(name string) error {
 			return derr.NewRequestNotFoundError(fmt.Errorf("Get %s: no such volume", name))
 
 		case *storage.RemoveVolumeConflict:
-			return derr.NewRequestConflictError(fmt.Errorf("Volume '%s' is in use", name))
+			return derr.NewRequestConflictError(fmt.Errorf(err.Payload.Message))
```

Comment:
Why switch this to the message we get from the Portlayer? Since this error return will need to be specific to docker.